action.yml

```diff
@@ -62,6 +87,7 @@ runs:
           git add -A .
           git commit -m "update reference docs to cli@${{ inputs.client-branch }}, core@${{ inputs.core-branch }}"
           git remote set-url origin https://${{ inputs.access-token }}@github.com/wandb/gitbook.git ; \
+          git pull ; \
```

Comment:
Why is this necessary now?

Reply:
in case new changes have been made from another source, since there's gonna be two different projects updating the docs now

Reply:
as in "while this action has been running in the current repo"? yea, makes sense, thnx.